apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/controller/ItemController.java

```diff
@@ -33,22 +33,13 @@
 import com.ctrip.framework.apollo.common.utils.BeanUtils;
 import com.ctrip.framework.apollo.core.utils.StringUtils;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Objects;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
```

Comment:
I think use Single-Type-Import (e.g., import java.util.List) increases the readability of the program and it will be very clear which classes have been used. 

BTW, in compile time, the import java.util.* will search for whole package to find the correct class' information.

Reply:
ok. my idea transfer import * when importing more than 5 classes in one package ,  I'll increase it.